apps/meteor/client/sidebar/hooks/useQueryOptions.ts

```diff
@@ -13,20 +14,35 @@ export const useQueryOptions = (): {
 		| {
 				lowerCaseName: number;
 				lm?: number | undefined;
+		  }
+		| {
+				priorityWeight: number;
+				estimatedServiceTimeAt: number;
+				ts: number;
 		  };
-} => {
-	const sortBy = useUserPreference('sidebarSortby');
-	const showRealName = useSetting('UI_Use_Real_Name');
+};
+
+const getSortOption = (sortingMechanism: OmniSortType, sortBy: string, showRealName: boolean) =>
+	({
+		[OmniSortType.Priority]: { priorityWeight: 1, estimatedServiceTimeAt: 1, ts: 1 },
+		[OmniSortType.SLAs]: { estimatedServiceTimeAt: 1, priorityWeight: 1, ts: 1 },
```

Comment:
For priorities and SLAs, can you please use this helper function instead to get the sort options? Just so we have a single definition of it
https://github.com/RocketChat/Rocket.Chat/blob/5b6a6962f9754136900ad431a86005faa3558525/apps/meteor/app/livechat/lib/inquiries.ts#L9